source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,183 @@
+
+Muting and unmuting categories
+------------------------------
+
+When you mute or unmute a category, all channels within that category are also muted or unmuted. You can still selectively unmute specific channels within a muted category.
+
+Select the **Category options** icon in the sidebar, then select **Mute Category**.
+
+Once a category is muted:
+
+- Email, desktop, and push notifications are disabled for all channels in the category.
+- A mute icon displays next to each channel name in the category.
+- The category and all of its channels appear at reduced opacity in the left-hand sidebar. Channels in the category aren't marked as unread unless you’re mentioned directly.
+
+To unmute the category, select the **Category options** icon in the sidebar, then select **Unmute Category**.
+
+    .. image:: ../../images/mute-categories.gif
+
+Sorting channels in categories
+------------------------------
+
+Select the **Category options** icon in the sidebar, then select **Sort** and choose from **Alphabetically**, **Recent Activity**, or **Manually**.
+
+    .. image:: ../../images/sort-categories.gif
+
+Group unread channels separately
+--------------------------------
+
+By default, Mattermost provides a one-click **Unreads** filter to only show channels with unread activity. Alternatively, you may choose to automatically group unread channels in their own category at the top of your sidebar.
+
+Go to **Main Menu > Account Settings > Sidebar**, set **Group unread channels separately** to **On**, then select **Save**.
+
+- When this setting is enabled, all unread messages appear only in the **Unreads** category, sorted with mentions first.
+- When this setting is disabled, all unread messages appear within their respective categories and channels. You can use the **Unread filter** to focus on only unread channels in the sidebar.
+
+When enabled, unread channels with mentions will sort to the top of the category.
+
+    .. image:: ../../images/unreads.gif
+
+.. tip::
+  
+  If you prefer to see only unread channels in their respective categories, we recommend collapsing your custom categories and disabling **Group unread channels separately** under **Account Settings > Sidebar**.
+
+Managing Direct Messages
+------------------------
+
+To sort your Direct Messages, select the **Channel options** icon in the sidebar, then select **Sort** and choose from **Alphabetically** or **Recent Activity**.
+
+How many Direct Messages to display?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Control how many Direct Message conversations display in the **Direct Messages** category to keep your conversations manageable. You can choose to show all messages or a fixed number of messages.
+
+You have two ways to configure the number of Direct Messages to display:
+
+- Go to **Main Menu > Account Settings > Sidebar**, then set **Number of direct messages to show**. Or select the **Channel options** icon in the sidebar, then select **Show**.
```

Comment:
Same feedback as for the previous instance of this: Indicating that there are two options should result in two separate options rather than a single sentence. The bullet should also be removed imo.

Alternatively, this can remain as-is, but the preceding sentence should be reworded to: "You can configure the number of Direct Messages to display:".

Reply:
Good feedback, @justinegeffen . We'll want to review this convention against the rest of the UG content and identify a good path forward.